angular-workspace/package.json

```diff
@@ -57,6 +57,6 @@
     "ng-packagr": "^14.1.0",
     "playwright": "^1.30.0",
     "rollup": "^3.10.1",
-    "typescript": "~4.7.4"
+    "typescript": "~4.8.0"
```

Comment:
If we mark it as "~4.8.2" everywhere that'll make it so we are in a supported typescript version range for Angular >=14.2 and <16
Looks like the lock file is resolving to 4.8.4 so shouldn't need retesting, but will prevent any weirdness accidentally dropping to less than 4.8.2
https://angular.io/guide/versions